test/integration/apimachinery/BUILD

```diff
@@ -2,6 +2,7 @@ load("@io_bazel_rules_go//go:def.bzl", "go_test")
 
 go_test(
     name = "go_default_test",
+    size = "large",
```

Comment:
I think a better solution is
    timeout = "long"

I found the following in the doc https://docs.bazel.build/versions/master/test-encyclopedia.html:
"Unlike timeout, the size also determines the limits for other resources like RAM, as described in Common definitions."

Reply:
I think another reason is more test cases, that is why I set is to 'large'

Reply:
Actually, there is only one test case in apimachinery integ